### DISCUSSION TITLE
`rest_framework.fields.SlugField` have bug

### DISCUSSION THREAD

**Discussion by xuxinwen** [encode/django-rest-framework#7763] [Potential Issue]
## Checklist

- [x] I have verified that that issue exists against the `master` branch of Django REST framework.
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] This is not a usage question. (Those should be directed to the [discussion group](https://groups.google.com/forum/#!forum/django-rest-framework) instead.)
- [x] This cannot be dealt with as a third party library. (We prefer new functionality to be [in the form of third party libraries](https://www.django-rest-framework.org/community/third-party-packages/#about-third-party-packages) where possible.)
- [x] I have reduced the issue to the simplest possible case.
- [x] I have included a failing test as a pull request. (If you are unable to do so we can still accept the issue.)

## Steps to reproduce
1. I have defined `Platform` model and Its *name*  field type is `SlugField`  
![image](https://user-images.githubusercontent.com/8899458/105118838-18466800-5b0a-11eb-8811-dfd3443d82cb.png)
2. I have defined `PlatformSerializer` serializer mapping to `Platform`
![image](https://user-images.githubusercontent.com/8899458/105118965-5774b900-5b0a-11eb-8a14-6c18e02097a5.png)

## Expected behavior
`PlatformSerializer.name.validators` only have one `RegexValidator` validator

## Actual behavior
1. I find `PlatformSerializer.name.validators` have two `RegexValidator` validators
![image](https://user-images.githubusercontent.com/8899458/105119449-509a7600-5b0b-11eb-82d1-6b54ee65271d.png)
2. This will lead to serializer error messages duplicate
![image](https://user-images.githubusercontent.com/8899458/105119538-7d4e8d80-5b0b-11eb-8158-184df5498c99.png)

**Comment by ahsanshafiq742** [comment]
Hey @auvipy @xuxinwen 
I would like to work on this bug, How do i assign this issue to me?

**Comment by auvipy** [comment]
> Hey @auvipy @xuxinwen
> I would like to work on this bug, How do i assign this issue to me?

I'm an external contributor. you can try and come with a PR
